core/spec/rails_helper.rb

```diff
@@ -41,7 +42,8 @@
     Rails.cache.clear
   end
 
+  config.include Spree::TestingSupport::JobHelpers
+
   config.include ActiveSupport::Testing::Assertions
```

Comment:
I think that without `ActiveJob::TestHelper` we can get rid of `ActiveSupport::Testing::Assertions` too